src/pages/settings-page/components/ProfileEdit/hooks/use-send-data.ts

```diff
@@ -17,37 +20,55 @@ interface IDataToSend {
   experience: ExperienceAndEducation[];
   profileSkills: string[];
 }
+type NotificationType = 'success' | 'error';
 
 export const useSendData = () => {
+  const { t } = useTranslation();
+  const [sendProfile] = usePostProfileMutation();
+  const openNotificationWithIcon = (type: NotificationType) => {
+    notification[type]({
+      message: type === 'success' ? `${t('ContactInfo.success')}` : `${t('ContactInfo.error')}`,
+      description:
+        type === 'success'
+          ? `${t('ContactInfo.dataHasBeenSaved')}`
+          : `${t('ContactInfo.someErrorOccurred')}`,
+    });
+  };
```

Comment:
I see that this code is duplicated through different components, does it make sense do a wrapper or something like that?